openstack/networking/v2/extensions/attributestag/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
+Package attributestag manages Tags on Resources created by the OpenStack Neutron Service.
+
+This enables tagging via a standard interface for resources types which support it.
+
+See https://developer.openstack.org/api-ref/network/v2/#standard-attributes-tag-extension for more information on the underlying API.
+
+Example to ReplaceAll Resource Tags
+
+    network, err := networks.Create(conn, createOpts).Extract()
+
+    tagReplaceAllOpts := attributestag.ReplaceAllOpts{
+        Tags:         []string{"abc", "123"},
+    }
+    attributestag.ReplaceAll(conn, "networks", network.ID, tagReplaceAllOpts)
+
+
+*/
+package attributestag
```

Comment:
I'm _really_ sorry, but this will bother me forever otherwise: can we call this `attributestags`?